pydecred/rpc.py

```diff
@@ -60,6 +60,163 @@ def getBlockchainInfo(self):
         """
         return GetBlockChainInfoResult.parse(self.call("getblockchaininfo"))
 
+    def getRawTransaction(self, txid, verbose=0):
+        """
+        Returns information about a transaction given its hash.
+
+        Args:
+            txid (str): The hash of the transaction
+            verbose (int): Optional. Default=0. Specifies the transaction is
+                returned as a JSON object instead of a hex-encoded string
+
+        Returns:
+            RawTransactionResult or string: RawTransactionResult if verbose=1,
+                hex string for the transaction if default
```

Comment:
Let's do a `msgtx.MsgTx` here, which is closer to the dcrd `dcrutil.Tx` returned by `rpcclient.Client`. Or you could just copy the `dcrutil.Tx` type.